tests/contrib/flask/test_flask_snapshot.py

```diff
@@ -63,16 +83,16 @@ def flask_client(flask_command, flask_env, flask_port):
         stdout=subprocess.PIPE,
         stderr=subprocess.PIPE,
         close_fds=True,
-        env=flask_env,
+        env=flask_env_arg(flask_wsgi_application),
         preexec_fn=os.setsid,
+        cwd=str(os.path.dirname(os.path.dirname(os.path.dirname(os.path.dirname(__file__))))),
```

Comment:
This was needed for this to work on my machine. Even if it worked previously on CI, it's more "correct" so I would like to leave it there even if not totally needed for this PR.